tests/unit_test/data_processor/action_serializer_test.py

```
@@ -0,0 +1,601 @@
+    "bot": bot_id_download
+}
+
+@pytest.fixture(autouse=True, scope='class')
+def setup():
+    connect(**Utility.mongoengine_connection(Utility.environment['database']["url"]))
+
+
+def test_get_item_name():
+    assert ActionSerializer.get_item_name(valid_http_action_config) == "a_api_action"
+    assert ActionSerializer.get_item_name(valid_pyscript_action_config) == "a_pyscript_action"
+    with pytest.raises(Exception):
```

Ruff

`pytest.raises(Exception)` should be considered evil

(B017)